packages/reverse-proxy-service/src/server.ts

```diff
@@ -12,6 +12,13 @@ import store from './setup/store';
 import { COOKIE_SECRET } from './setup/env';
 import oidcAuth from './middleware/oidcAuth';
 import { updateApplicationCache } from './utils/applicationCache';
+import http from 'http';
+
+declare module 'http' { interface Agent { maxHeaderSize?: number; } }
```

Comment:
Your types should be n the global declation files. not to be mixed with code.

Reply:
Hey @riginoommen , could you please check now !